src/main/java/com/minecolonies/coremod/entity/ai/citizen/deliveryman/EntityAIWorkDeliveryman.java

```diff
@@ -129,8 +132,20 @@ public EntityAIWorkDeliveryman(@NotNull final JobDeliveryman deliveryman)
      */
     public AIState gather()
     {
+        if (maximalGatherCount < 0)
+        {
+            maximalGatherCount = Configurations.requestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.requestSystem.maximalBuildingsToGather - Configurations.requestSystem.minimalBuildingsToGather));
+        }
+
         if (gatherTarget == null)
         {
+            if (gatherCount == maximalGatherCount)
+            {
+                maximalGatherCount = Configurations.requestSystem.minimalBuildingsToGather + worker.getRandom().nextInt(Math.max(1, Configurations.requestSystem.maximalBuildingsToGather - Configurations.requestSystem.minimalBuildingsToGather));
+                gatherCount = 0;
+                return DUMPING;
+            }
+
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Split this 244 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS00103)